pmm-app/tests/verifyPMMInventory_test.js

```diff
@@ -154,7 +154,8 @@ Scenario(
     const countOfRunning = await pmmInventoryPage.getCountOfRunningAgents();
     // Need countOfPMMAgentType because agents that have Type PMM Agent don't have status. We need subtract it
     const countOfPMMAgentType = await pmmInventoryPage.getCountOfPMMAgents();
+    const serviceIdsNotRunning = await pmmInventoryPage.getNotRunningAgentsServiceId();
```

Comment:
So we are sure we will always have Agents in Waiting, Starting and Unknown status?

Reply:
I've changed it to the array of statuses. We can easily delete or add new status in the future if it will be needed. :)